tests/test_api_request_handler_labels.py

```
@@ -0,0 +1,1075 @@
+            assert result is None
+
+    def test_add_label_multipart_format(self, labels_handler):
+        """Test label addition uses multipart/form-data format"""
+        mock_response = APIClientResult(
+            status_code=200,
+            response_text={"id": 1, "title": "Test Label"},
+            error_message=None
+        )
+        
+        with patch.object(labels_handler.client, 'send_post', return_value=mock_response):
+            result, error = labels_handler.add_label(project_id=1, title="Test Label")
```

Ruff (0.14.3)

Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)